docs/docs/clients/overview.md

```diff
@@ -120,34 +121,76 @@ running in `Local Evaluation` mode.
 
 :::
 
-## Pros, Cons and Caveats
+## Networking Model
 
-### Remote Evaluation Mode
+When are network requests made, and when do you need to consider network latency? It depends on your evaluation mode,
+and whether you are using Client-side or Server-side SDKs!
 
-- Identities are persisted within the Flagsmith Datastore.
-- Identity overrides specified within the Dashboard.
-- All Integrations work as designed.
+### Remote Evaluation
 
-### Local Evaluation Mode
+#### Client-side
 
-- Identities are _not_ sent to the API and so are not persisted in the datastore.
-- Because Local mode does not connect to the datastore for each Flag request, it is not able to read the Trait data of
-  Identities from the API. This means that you have to provide the full complement of Traits when requesting the Flags
-  for a particular Identity. Our SDKs all provide relevant methods to achieve this.
-- [Identity overrides](../basic-features/managing-identities#identity-overrides) do not operate at all.
-- Analytics-based Integrations do not run.
+- Flagsmith SDKs will store all the Flags for an Environment in local memory. Flag evaluations within the SDK are then a
```

Comment:
This isn't very clear - it doesn't _magically_ store all the flags in local memory. It does it following the first network request. I don't think it _always_ does it either - there are configuration parameters for each of the clients to configure the cache etc.

Reply:
OK tweaked. Its just a general overview but I think this info is important at this stage in the docs.